scripts/ci/rust_checks.py

```diff
@@ -79,7 +80,20 @@ def main() -> None:
     timings.append(run_cargo("check", "-p rerun --no-default-features"))
     timings.append(run_cargo("check", "-p rerun --no-default-features --features sdk"))
 
-    # Since features are additive, check crates individually.
+    if args.skip_wasm_checks is not True:
```

Comment:
forgot to actually use this option! I figured we don't have to run this on pr ci